src/vma/dev/ring_slave.cpp

```diff
@@ -222,6 +224,22 @@ void ring_slave::flow_tcp_del_all()
 	}
 }
 
+bool ring_slave::request_more_buffers(descq_t &local_pool, uint32_t count, uint32_t lkey, bool is_rx)
```

Comment:
Please consider two inline functions as `request_more_rx_buffers` and `request_more_tx_buffers`
It is improve code readability and exclude if condition

Reply:
i reverted this commit